interface/backend/static/context_processors.py

```diff
@@ -0,0 +1,5 @@
+from django.conf import settings
+
+
+def git_version(request):
+    return {'APP_VERSION_NUMBER': settings.APP_VERSION_NUMBER}
```

Comment:
Did you see my idea about just including ``django.conf.settings`` in a ``settings_context`` ? That prevents us having to add other context processors in the future :)